bugwarrior/docs/services/gmail.rst

```diff
@@ -41,6 +42,9 @@ You do not need to specify the ``login_name``, but it can be useful to avoid
 accidentally fetching data from the wrong account. (This also allows multiple
 targets with the same login to share the same authentication token.)
 
+By default, the last 100 threads matching the query will be returned. You may
+adjust the number of returned threads with ``gmail.thread_limit``.
```

Comment:
For consistency with other services, the Example Service section only includes essential fields and optional fields go in their own section with an example of changing from the default behavior.

```suggestion
Thread Limit
-----------------

Gmail search results are capped at 100 max results by default, but may be
adjusted to meet your needs by specifying `gmail.email_limit`:

.. config::
    :fragment: gmail
    
    gmail.thread_limit = 1000
```